app/server/api/user_api.py

```diff
@@ -141,55 +136,57 @@ def post(self, user_id):
 
         return make_response(jsonify(response_object)), response_code
 
-    @requires_auth(allowed_roles={'ADMIN': 'subadmin'})
-    def put(self, user_id):
-        put_data = request.get_json()
 
-        first_name = put_data.get('first_name')
-        last_name = put_data.get('last_name')
+@requires_auth(allowed_roles={'ADMIN': 'subadmin'})
+def put(self, user_id):
+    put_data = request.get_json()
 
-        email = put_data.get('email')
-        phone = put_data.get('phone')
-        public_serial_number = put_data.get('public_serial_number')
-        location = put_data.get('location')
+    first_name = put_data.get('first_name')
+    last_name = put_data.get('last_name')
 
-        user = User.query.get(user_id)
+    email = put_data.get('email')
+    phone = put_data.get('phone')
+    public_serial_number = put_data.get('public_serial_number')
+    location = put_data.get('location')
 
-        if not user:
-            response_object = {
-                'message': 'User not found'
-            }
-            return make_response(jsonify(response_object)), 400
+    user = User.query.get(user_id)
 
-        if first_name and not first_name == user.first_name:
-            user.first_name = first_name
+    if not user:
+        response_object = {
+            'message': 'User not found'
+        }
+        return make_response(jsonify(response_object)), 400
 
-        if last_name and not last_name == user.last_name:
-            user.last_name = last_name
+    if first_name and not first_name == user.first_name:
```

Comment:
I think my linter auto styled some stuff. I think it may be worth it to switch to pycharm for me

Reply:
we may want some sort of linter config file to enforce one set of style guide, so we don't end up with so many of these commits changing styles back and forth (mine also did this)

Reply:
I think linter config is a good idea. @tristanhcole and my pycharm settings might have something really funky with tabs and spaces, because the resulting output isn't just a style change - it's pulling the entire `put` method out of the class! 😱